packages/arb-token-bridge-ui/src/components/TransferPanel/TokenImportDialog.tsx

```diff
@@ -95,6 +102,11 @@ export function TokenImportDialog({
     l2Provider: childChainProvider
   })
 
+  const isTeleportMode = useTeleportMode({
```

Comment:
Actually I wonder if we could have it in `useNetworksRelationship` instead? Looks like a good place for it without the need for another hook.